gitlab/resource_gitlab_user_impersonation_token.go

```diff
@@ -0,0 +1,137 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
```

Comment:
Please upgrade to the [terraform SDK v2](https://www.terraform.io/plugin/sdkv2/sdkv2-intro) and make sure to rebase to an up-to-date `master` branch.